R/dvhydrograph-data.R

```diff
@@ -103,6 +105,21 @@ parseRefData <- function(data, series) {
   return(plotData)
 }
 
+parseEstimatedEdges <- function(data, allVars){
```

Comment:
add description & argument definitions to this (trying to be better about documentation whenever we add new functions)